specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/virtualWan.json

```diff
@@ -6884,6 +6884,10 @@
           "type": "boolean",
           "description": "Enable internet security."
         },
+        "expressRouteGatewayBypass": {
+          "type": "boolean",
+          "description": "Enable FastPath to vWan Firewall hub."
```

Comment:
seeing all examples are at false, do you want to add `"default": false`

Reply:
I have kept the default setting same as this flag used in a different resource. https://github.com/Azure/azure-rest-api-specs/blob/master/specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/virtualNetworkGateway.json#L2986
Let me get back on this.

Reply:
@allenjzhang Keeping the setting as the flag used in the other resource. We can merger this now 👍